src/install/haproxy/appInfo.mdx

```diff
@@ -0,0 +1,9 @@
+---
+componentType: appInfoConfig
+headingText: Choose your framework
```

Comment:
Does anything change if they choose Windows or Linux? (e.g. Windows with anything not MSI?) I see a change when they select an install path, but want to make sure i'm not missing anything... Assuming Windows is only relevant for MSI so choosing Windows > Linux path wouldn't actually matter since it's not a ~thing